rest-api/resources/src/main/resources/openapi/dataMessage/dataMessage-scopeId.yaml

```diff
@@ -67,6 +67,15 @@ paths:
           in: query
           schema:
             type: string
+        - description: The sort direction. Can be ASCENDING, DESCENDING (default). Case-insensitive.
```

Comment:
Whoops... this slipped in as well. `ASC` and `DESC` here too. Just one to go!

Reply:
It is a general description, those two have the general meaning.

Reply:
This is a generic description and the meaning of those two words is generic as well.
They are not connected to actual enum members.

Reply:
Hi! 

That makes sense. Since they are generic description maybe those should be lowercase? So they will not be confused as values that can be used!

Thanks,

_Alberto_

Reply:
I agree with @Coduz. I would rather write them lowercase, and even remove the reference for them being case insensitive.